sshuttle/options.py

```diff
@@ -38,6 +40,15 @@ def parse_subnetport(s):
         raise Fatal('%r is not a valid address/mask:port format' % s)
 
     addr, width, fport, lport = m.groups()
+    # Attempt to 0 pad partial IPv4 address.
+    # 10 -> 10.0.0.0
+    if not ipv6:
+        try:
+            if addr.count('.') > 0 or int(addr) < 256:
```

Comment:
This is a bit gross, but the only reasonable way I could figure out how to maintain current behavior to allow passing in `localhost` or an IP address as an integer, which is tested as `3098282570`.